src/networking.c

```diff
@@ -363,8 +363,14 @@ subnet_to_prefix(const gint family, const void *const sin_addr) {
 	switch(family) {
 	case AF_INET6:
 		pfix = prefix(addr, 16);
+		break;
 	case AF_INET:
 		pfix = prefix(addr, 4);
+		break;
+	default:
+		g_debug("invalid prefix family %d", family);
+		pfix = 0;
+		break;
```

Comment:
The linux kernel coding style allows for break here and in some cases is useful.